### ISSUE TITLE
Workspace.eachEditor is deprecated.

### ISSUE THREAD

**Issue by hank-harvey** [mpeterson2/save-session#98]
Use Workspace::observeTextEditors instead
```
Workspace.eachEditor (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:355:7)
Object.save (/Users/hankharvey/.atom/packages/save-session/lib/files.coffee:22:19)
```

**Comment by anthonyraymond** [comment]
Deplicate of #97

**Comment by mpeterson2** [comment]
Should be fixed in `0.13.0`
